### DISCUSSION TITLE
How to stop Logdy?

### DISCUSSION THREAD

**Discussion by DaveTSG** [logdyhq/logdy-core#45] [Q&A] [answered]
Probably a stupid question, but how do you stop Logdy from running?

I was following [the quick start guide](https://logdy.dev/docs/quick-start), ran the installation script, then ran `echo "test" | logdy`. That fired up the server and I can see "test" in the logs in the Logdy UI at http://127.0.0.1:8080/

However, I then used Ctrl+C to stop Logdy... but the web server is still running and accessible.

I tried `ps aux | grep "logdy"`, thinking I would find the process ID and kill it, but all I got was the grep process itself, not logdy, so I can't kill the process...

I also ran `logdy help` but there was nothing in there that made it obvious how to stop it.

I ended up rebooting the machine and that did the trick, but is there a better way of stopping Logdy?

Thanks in advance.

**Comment by PeterOsinski** [comment]
Hey @DaveTSG if you cannot find logdy in running processes (`ps aux`) then it's not possible for the webserver to be working. I tried multiple different ways of invoking and killing the process, but it always exited. Are you able to reproduce the issue?

**Comment by DaveTSG** [comment, marked as answer]
I realise it makes no sense, but despite not showing up in `ps aux`, the web server definitely was still accessible, even after a few reboots. Eventually, I uninstalled logdy completely, and that solved the problem.
